Dockerfile

```diff
@@ -1,25 +1,31 @@
+FROM maven:3.6.1-jdk-8 AS tls-scanner-build
+
+RUN git clone --depth=1 https://github.com/RUB-NDS/TLS-Attacker.git && \
```

Comment:
Sorry, currently we need TLS-Attacker version 3.0: https://github.com/RUB-NDS/TLS-Attacker/tree/3.0

Because of the TLS-Scanner 2.9 requirements, check out the next commentary.

Reply:
I fixed it. To be precise, TLS-Scanner 2.9 requires TLS-Attacker 3.1. Now it should work.